services/analytics/profile.go

```diff
@@ -160,6 +160,11 @@ func (s *Service) GetReport(ctx context.Context, in *qanpb.ReportRequest) (*qanp
 			row.Fingerprint = "TOTAL"
 		}
 
+		isTotal := false
+		if i == 0 {
```

Comment:
@askomorokhov Maybe is better to use isTotal := i == 0 here. What do you think?

Reply:
Changed.

Reply:
Thanks, approved.